### PR TITLE
Add `[p]cc raw` command giving raw (escaped) response

### PR DESCRIPTION
**Pull request by NeuroAssassin**
### Type

- [ ] Bugfix
- [X] Enhancement
- [X] New feature

### Description of the changes
Add `[p]cc raw`

### CODE REVIEW

File: redbot/cogs/customcom/customcom.py [hunk 1]
```diff
@@ -206,6 +206,24 @@ async def customcom(self, ctx: commands.Context):
         """Custom commands management."""
         pass
 
+    @customcom.command(name="raw")
+    async def cc_raw(self, ctx: commands.Context, command: str.lower):
+        """Get the raw response of a custom command, to get the proper markdown.
+        
+        This is helpful for copy and pasting."""
+        commands = await self.config.guild(ctx.guild).commands()
+        if not command in commands:
```
Comment (Jackenmen) [inline redbot/cogs/customcom/customcom.py]: You should use `x not in y` over `not x in y`:
```suggestion
        if command not in commands:
```

File: redbot/cogs/customcom/customcom.py [hunk 2]
```diff
@@ -206,6 +206,24 @@ async def customcom(self, ctx: commands.Context):
...
+            return await ctx.send("That command doesn't exist.")
+        command = commands[command]
+        if type(command["response"]) == str:
```
Comment (Jackenmen) [inline redbot/cogs/customcom/customcom.py]:
```suggestion
        if isinstance(command["response"], str):
```

File: redbot/cogs/customcom/customcom.py [hunk 3]
```diff
@@ -206,6 +206,24 @@ async def customcom(self, ctx: commands.Context):
...
+            raw = discord.utils.escape_markdown(command["response"])
+            await ctx.send(raw)
+        else:
+            await ctx.send(_("The following are responses recorded:"))
+            for number, response in enumerate(command["response"], start=1):
+                raw = discord.utils.escape_markdown(response)
+                await ctx.send(f"{str(number)} - {raw}")
```
Comment (Jackenmen) [inline redbot/cogs/customcom/customcom.py]: I think this would be a lot better with menus

**Review by Jackenmen — CHANGES_REQUESTED** [review]

**Comment by NeuroAssassin** [comment]
All changes addressed

File: redbot/cogs/customcom/customcom.py [hunk 4]
```diff
@@ -206,6 +206,39 @@ async def customcom(self, ctx: commands.Context):
...
+            raw = discord.utils.escape_markdown(command["response"])
+            await ctx.send(raw)
+        else:
+            msglist = []
+            if await ctx.embed_requested():
+                colour = await ctx.embed_colour()
+                for number, response in enumerate(command["response"], start=1):
+                    raw = discord.utils.escape_markdown(response)
+                    embed = discord.Embed(
+                        title=_("Response #{num}/{total}".format(num=number, total=len(command["response"]))),
```
Comment (jack1142) [inline redbot/cogs/customcom/customcom.py]: String needs to be formatted *after* translation.
```suggestion
                        title=_("Response #{num}/{total}").format(num=number, total=len(command["response"])),
```

File: redbot/cogs/customcom/customcom.py [hunk 5]
```diff
@@ -206,6 +206,39 @@ async def customcom(self, ctx: commands.Context):
...
+                        description=raw,
+                        colour=colour
+                    )
+                    msglist.append(embed)
+            else:
+                for number, response in enumerate(command["response"], start=1):
+                    raw = discord.utils.escape_markdown(response)
+                    msg = _(
+                        "Response #{num}/{total}:\n{raw}".format(num=number, total=len(command["response"]), raw=raw)
+                    )
```
Comment (jack1142) [inline redbot/cogs/customcom/customcom.py]: String needs to be formatted *after* translation.
```suggestion
                    msg = _("Response #{num}/{total}:\n{raw}").format(
                        num=number, total=len(command["response"]), raw=raw
                    )
```

File: redbot/cogs/customcom/customcom.py [hunk 6]
```diff
@@ -206,6 +206,41 @@ async def customcom(self, ctx: commands.Context):
...
+                    msglist.append(embed)
+            else:
+                for number, response in enumerate(command["response"], start=1):
+                    raw = discord.utils.escape_markdown(response)
+                    msg = _(
+                        "Response #{num}/{total}:\n{raw}".format(
+                            num=number, total=len(command["response"]), raw=raw
+                        )
+                    )
```
Comment (Jackenmen) [inline redbot/cogs/customcom/customcom.py]: String needs to be formatted *after* translation.
This variant might also suffer for an edge-case where the response is close to discord message char limit (2k) though I haven't tested that. The embed variant isn't affected cause embed's description can be up to 2048 characters and embed's title doesn't count into that.
```suggestion
                    msg = _("Response #{num}/{total}:\n{raw}").format(
                        num=number, total=len(command["response"]), raw=raw
                    )
```

File: redbot/cogs/customcom/customcom.py [hunk 7]
```diff
@@ -206,6 +206,45 @@ async def customcom(self, ctx: commands.Context):
...
+            if await ctx.embed_requested():
+                colour = await ctx.embed_colour()
+                for number, response in enumerate(command["response"], start=1):
+                    raw = discord.utils.escape_markdown(response)
+                    embed = discord.Embed(
+                        title=_(
+                            "Response #{num}/{total}".format(
+                                num=number, total=len(command["response"])
+                            )
+                        ),
```
Comment (Jackenmen) [inline redbot/cogs/customcom/customcom.py]: String needs to be formatted *after* translation.
```suggestion
                        title=_("Response #{num}/{total}").format(
                            num=number, total=len(command["response"])
                        ),
```

**Review by Jackenmen — CHANGES_REQUESTED** [review]

**Review by Jackenmen — APPROVED** [review]
LGTM

**Pull request merged**